### PR TITLE
Remove references to "node" in EntityAliasTypeBase

### PR DESCRIPTION
**Pull request by randomletters**
Replace them with "entity".

### CODE REVIEW

**Comment by randomletters** [comment]
I do not believe the test failures have anything to do with my edit.  The only user facing change was changing the words "node, nodes" to "entity, entities" which does not appear to be the cause of the failure.

File: src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php [hunk 1]
```diff
@@ -267,12 +267,12 @@ protected function bulkUpdate(array $ids, array $options = array()) {
...
     $entities = $this->entityManager->getStorage($this->getEntityTypeId())->loadMultiple($ids);
-    foreach ($entities as $node) {
-      \Drupal::service('pathauto.manager')->updateAlias($node, 'bulkupdate', $options);
+    foreach ($entities as $entity) {
+      \Drupal::service('pathauto.manager')->updateAlias($entity, 'bulkupdate', $options);
     }
 
     if (!empty($options['message'])) {
-      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 node.', 'Updated URL aliases for @count nodes.'));
+      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 entity.', 'Updated URL aliases for @count entities.'));
```
Comment (Berdir) [inline src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php]: We shouldn't hardcode entity either.. the problem is that there is no generic plural entity label. You could try to rewrite it into something like "for type %label" or so...

**Comment by randomletters** [comment]
I updated the wording based on your suggestion.

File: src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php [hunk 2]
```diff
@@ -267,12 +267,12 @@ protected function bulkUpdate(array $ids, array $options = array()) {
...
     $entities = $this->entityManager->getStorage($this->getEntityTypeId())->loadMultiple($ids);
-    foreach ($entities as $node) {
-      \Drupal::service('pathauto.manager')->updateAlias($node, 'bulkupdate', $options);
+    foreach ($entities as $entity) {
+      \Drupal::service('pathauto.manager')->updateAlias($entity, 'bulkupdate', $options);
     }
 
     if (!empty($options['message'])) {
-      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated URL alias for 1 node.', 'Updated URL aliases for @count nodes.'));
+      drupal_set_message(\Drupal::translation()->formatPlural(count($ids), 'Updated 1 URL alias for type %label.', 'Updated @count URL aliases for type %label.'), array('%label' => $entity->getLabel()));
```
Comment (Berdir) [inline src/Plugin/pathauto/AliasType/EntityAliasTypeBase.php]: $entity->getLabel() is e.g. a specific node title, $entity is the last entity from the loop above, that's not what you want. Instead, you need to get the entity type definition from the entity manager and then the label of that.

**Comment by randomletters** [comment]
I changed the label to $this->getLabel() and made the wording of the message less awkward.  This should result in messages similar to the following:

Updated 1 Content URL alias.
Updated 4 User URL aliases.
Updated 2 Taxonomy term paths URL aliases.

The last one is slightly awkward because the label for the taxonomy term alias plugin is inconsistent.  I think it should be changed to "Taxonomy term" or at least kept singular "Taxonomy term path".

**Comment by Berdir** [comment]
Works for me, merged.

**Pull request merged**